codewof/programming/content/en/evens-out/question.md

```diff
@@ -0,0 +1,7 @@
+# Evens Out
+
+The following function `evens-out(numbers)` is supposed to take a list of numbers and **return** a new list containing only the odd numbers from the original list.
+
+But there's a problem: it is leaving out all the *odd* numbers, instead of all the evens.
+
```

Comment:
Could a comment be added mentioning that the % sign is called the modulo operator? Just to make it easier for them to google it if they want to.

Reply:
I'll add _Hint: the `%` operator is called the 'modulo' operator._
Does that work?

Reply:
Yep, that's great